lib/galaxy/job_metrics/instrumenters/cgroup.py

```diff
@@ -114,18 +114,18 @@
 
 
 class CgroupPluginFormatter(formatting.JobMetricFormatter):
-    def format(self, key, value):
+    def format(self, key: str, value: Any) -> formatting.FormattedMetric:
```

Comment:
The type annotation should be usually inherited from `JobMetricFormatter.format()`, so no need to add it here, but I think `JobMetricFormatter.format()` annotation should be changed to `key: str` instead of `key: Any` (as you are doing here).

Reply:
Might be that I'm leaning on typescript understanding too much here but until I annotated the return explicitly mypy wasn't flagging an error.  Is there a better way to enforce that the type annotation should be inherited?

Reply:
Ah, I think this is https://github.com/python/mypy/issues/9603 , i.e. even though mypy checks the content of an unannotated method (since we are using `check_untyped_defs = True`), it doesn't check that the inferred return type is compatible with the return type of the parent method.

Reply:
Cool, so I'll leave this in.